### ISSUE TITLE
ghosting on certain puzzles / color combinations

### ISSUE THREAD

**Issue by raisjn** [mrichards42/remarkable_puzzles#12]
i was playing around with blackbox, trying to get rid of ghosting (not sure if you noticed it?)

COVER is set to 0.5. if BALL is set to < 0.5, we get ghosting when removing a BALL. if BALL is set > 0.5, it looks fine. it seems that going from dark -> light -> dark has less ghosting, but i don't understand why yet

**Comment by mrichards42** [comment]
Yeah, I've noticed this. I assume it's a fundamental issue with using DU? I assume you either get no ghosting or fast updates, but not both, but I'd love to be wrong about that.

You're right, it does seem to be especially noticeable when you go from black -> gray. Like if you open help on just about any puzzle and close it, you can see the text ghosted wherever there's gray shading.

**Comment by raisjn** [comment]
for text ghosting, maybe we upgrade waveform when help dialog is closed, so next refresh has less ghosting
